ios/BetterRail/RNBetterRail.swift

```diff
@@ -109,4 +110,13 @@ class RNBetterRail: NSObject {
       }
     }
   }
+
+  @available(iOS 16.2, *)
+  @objc func activityAuthorizationInfo(_ blackString: String, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
```

Comment:
Why is the empty string needed?  


Also:
```suggestion
  @objc func activityAuthorizationInfo(_ emptyString: String, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
```

Reply:
`resolve` must be the second parameter or later, so we must do this hack